linux_os/guide/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/ansible/shared.yml

```diff
@@ -1,13 +1,32 @@
-# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle,multi_platform_ubuntu
 # reboot = false
 # strategy = configure
 # complexity = low
 # disruption = low
 {{{ ansible_instantiate_variables("var_audispd_remote_server") }}}
 {{% set audisp_config_file_path = audisp_conf_path + "/audisp-remote.conf" %}}
 
-- name: Make sure that a remote server is configured for Audispd
-  lineinfile:
+{{% if 'ubuntu' in product %}}
+{{% set audisp_config_plugin_path = audisp_conf_path + "/plugins.d/au-remote.conf" %}}
+- name: "{{{ rule_title }}} - Uncomment active for offloading to remote server"
+  ansible.builtin.lineinfile:
+    path: "{{{ audisp_config_plugin_path }}}"
+    regexp: ^(#.*)(active\s*=)
+    line: \2
+    backrefs: true
+
+- name: "{{{ rule_title }}} - Set active to true for offloading to remote server"
+  ansible.builtin.lineinfile:
+    path: "{{{ audisp_config_plugin_path }}}"
+    regexp: ^(.*)(active\s*=)(?!.*yes)
```

Comment:
The test scenarios for this rule, eg. `ubuntu_correct.pass.sh`, use a capital A in `Active`, however, this remediation and the rule description use small a in `active. The OVAL seems to correctly use a case-insensitive regex modifier. Could you investigate this discrepancy about the A case?

Reply:
Good catch! Thanks for the feedback, I will take a look at this

Reply:
Not sure how you would like to proceed since the STIG and man page seem to use `active` over `Active`. Changes in this PR seem to be recognized but should the tests align with what is shown in the STIG definition and man page?

* https://linux.die.net/man/8/audispd
* https://www.stigviewer.com/stig/canonical_ubuntu_20.04_lts/2021-03-23/finding/V-238306

Reply:
Thanks for confirming that these options are case insensitive. Based on that, I think that:
* in the rule description we can use whatever we want, ie. it can stay lowercase
* in the OVAL we should use case-insentive check, which we already have
* the Ansible and Bash remediations should be able to set the correct option value regardless of the case used in the file (it can change it to lowercase during the remediation if that's convenient for us)
* the test scenarios should be testing multiple different letter cases.

Reply:
the bash remediation makes use of `set_config_file` macro which has a `insensitive` parameter. Perhaps ansible needs the same parameter, as `ansible_set_config_file` doesn't have one.

Reply:
This is a good idea, I'll work on adding the insensitive capabilities for `ansible_set_config_file`. Should this change be here or should I open up a new PR?

Reply:
@dodys excellent idea!

@yunimoo If the change of the macro would be large it would be better to have it as a separate PR.